pyproject.toml

```
@@ -1,3 +1,51 @@
+[build-system]
+requires = ["setuptools>=60"]
```

Support for the PEP 621 metadata format was introduced in 61.0.0.

```suggestion
requires = ["setuptools>=61"]
```